### PR TITLE
Fix report crash by freezing SQLAlchemy and doclinks fix

### LINKED ISSUE #231: Dead link to documentation in README
The README links to this url http://www.clinicalgenomics.se/chanjo/ which 404s

### LINKED ISSUE #232: AttributeError: 'scoped_session' object has no attribute 'engine' when trying to visualize report in browser
To reproduce:
- install the repo in a conda env
- Populate a demo database with data (see [instructions](https://github.com/Clinical-Genomics/chanjo-report/blob/af4990c8e7143545d1c68ec4cac3801204fed359/Makefile#L12) contained in the makefile (just the part starting with chanjo blablabla)

- Note that chanjo itself connects normally and returns data, for instance by running the command: 
```chanjo -d mysql+pymysql://user:password@localhost:3306/chanjo4_demo  db samples```

image

- When instead one runs the command to generate a report: 
```chanjo -d mysql+pymysql://user:password@localhost:3306/chanjo4_demo report```

The server starts: 
image

But when one goes to the browser page indicated by the app, it crashed with the following error:
```
Traceback (most recent call last):
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/sqlalchemy/util/_collections.py", line 1008, in __call__
    return self.registry[key]
KeyError: <greenlet.greenlet object at 0x7fba8cbcfa90 (otid=0x7fba8c936680) current active started main>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 16, in index
    sample_objs = api.query(Sample).limit(20)
  File "<string>", line 2, in query
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/sqlalchemy/orm/scoping.py", line 23, in _proxied
    return self.registry()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/sqlalchemy/util/_collections.py", line 1010, in __call__
    return self.registry.setdefault(key, self.createfunc())
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask_sqlalchemy/__init__.py", line 771, in create_session
    return SignallingSession(self, **options)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask_sqlalchemy/__init__.py", line 155, in __init__
    bind = options.pop('bind', None) or db.engine
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask_alchy.py", line 41, in __getattr__
    return getattr(self.session, attr)
AttributeError: 'scoped_session' object has no attribute 'engine'
```

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #232 (Error creating chanjo report)
- Fix #231 (Wrong link to the documentation)

### **How to prepare for test**:
- Follow the instructions from the issue: https://github.com/Clinical-Genomics/chanjo/issues/232#issue-939479078 and make sure that you can reproduce the error using chanjo master branch.

### How to test:
- Switch to this branch and install again chanjo (pip install -e .)
- Repeat the procedure described in the issue and notice that the html report is now working

### Expected outcome:
- [x] That it works!

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ toolz
 ruamel.yaml
 importlib_metadata
 pymysql
+sqlalchemy==1.3.*
```
Comment (northwestwitch) [inline requirements.txt:9]: @mhkc this is the real fix to the report crash problem!

File: setup.py [hunk 2]
```diff
@@ -49,78 +49,68 @@ def run_tests(self):
...
-    name='chanjo',
-
+    name="chanjo",
     # Versions should comply with PEP440. For a discussion on
     # single-sourcing the version across setup.py and the project code,
     # see http://packaging.python.org/en/latest/tutorial.html#version
-    version='4.6.0',
-
-    description='Coverage analysis tool for clinical sequencing',
+    version="4.6.1",
```
Comment (northwestwitch) [inline setup.py:61]: This is the only line I have changed in this file.

**Comment by northwestwitch** [comment]
@moonso are you working by chance?

**Comment by northwestwitch** [comment]
Behavior installing master branch on a fresh conda env and launching the report: ⛔ 

image


image


Behavior installing this branch: 🆗 

image

image

File: setup.py [hunk 3]
```diff
@@ -4,27 +4,27 @@
...
-if sys.argv[-1] == 'publish':
-    os.system('python setup.py sdist bdist_wheel upload')
+if sys.argv[-1] == "publish":
+    os.system("python setup.py sdist bdist_wheel upload")
     sys.exit()
 
 
-def parse_reqs(req_path='./requirements.txt'):
+def parse_reqs(req_path="./requirements.txt"):
     """Recursively parse requirements from nested pip files."""
```
Comment (mikaell) [inline setup.py]: Bonus points for recursion 😃

**Review by mikaell — APPROVED** [review]
Looks good. (Maybe we should freeze every dependency some day -but I suppose that's gonna happen automatically in Docker installations).

**Comment by mhkc** [comment]
Its working with the chanjo-report container as well.

**Comment by northwestwitch** [comment]
Thanks @mhkc and @mikaell, I'm going to merge and release then! 🥳

**Comment by mhkc** [comment]
> 
> 
> Looks good. (Maybe we should freeze every dependency some day -but I suppose that's gonna happen automatically in Docker installations).

Yes but every time you build a new version of the container you pull the latest version of some packages.

**Pull request merged**